packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -589,3 +589,59 @@ taskDefinition.addContainer('TheContainer', {
   })
 });
 ```
+
+## Capacity Provider
+
+### Autoscaling Group Capacity Provider
+
+Amazon ECS capacity providers can use Auto Scaling groups to manage the Amazon EC2 instances registered to
+their clusters. You can use the managed scaling feature to have Amazon ECS manage the scale-in and scale-out
+actions of the Auto Scaling group or you can manage the scaling actions yourself. This enables you to effectively
+use cluster auto scaling.
+
+The following sample creates two Autoscaling Group Capacity Providers and register them to the cluster.
```

Comment:
> Thank you so much for contributing❤️ I believe this will be very good news for our ECS customers!

Thanks for the review. This PR is a little bit challenging. Thanks for staying with me. Let's keep it moving.

Reply:
> Should this be Auto Scaling Group?

According this document
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/asg-capacity-providers.html

Maybe `Auto Scaling group" ?